civis/futures.py

```diff
@@ -0,0 +1,148 @@
+from civis import APIClient
+from civis.base import FAILED, DONE
+from civis.response import Response
+from civis.polling import PollableResult
+
+try:
+    from pubnub.pubnub import PubNub
+    from pubnub.pnconfiguration import PNConfiguration, PNReconnectionPolicy
+    from pubnub.callbacks import SubscribeCallback
+    has_pubnub = True
+except ImportError:
+    has_pubnub = False
+
+_LONG_POLLING_INTERVAL = 9.5 * 60
```

Comment:
Could you add a comment about where this number comes from? It's that `pubnub` can recover from network outages of up to 10 minutes, but we need to ensure that we catch completions which happen in longer outages, correct?